packages/plugins/typescript/generic-sdk/src/visitor.ts

```diff
@@ -69,9 +72,12 @@ export class GenericSdkVisitor extends ClientSideBaseVisitor<RawGenericSdkPlugin
           o.node.variableDefinitions.length === 0 ||
           o.node.variableDefinitions.every(v => v.type.kind !== Kind.NON_NULL_TYPE || v.defaultValue);
         const returnType = usingObservable && o.operationType === 'Subscription' ? 'Observable' : 'Promise';
+        const resultData = this.config.rawRequest
+          ? `{ data: ${o.operationResultType}, errors: any }`
```

Comment:
```suggestion
          ? `{ data?: ${o.operationResultType}, errors?: Array<{ message: string; extensions?: unknown }>, extensions?: unknown }`
```

Shouldn't this be the following instead?

Reply:
Added tests and updated this line as well